frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/base/built_in_page_widget.dart

```diff
@@ -53,11 +53,16 @@ class _BuiltInPageWidgetState extends State<BuiltInPageWidget> {
           final board = snapshot.data?.getLeftOrNull<ViewPB>();
           if (board != null) {
             return _build(context, board);
+          } else {
+            return const SizedBox.shrink();
           }
```

Comment:
I'd drop the `else` here since it's redundant. See [this link](https://stackoverflow.com/questions/3261849/why-is-else-rarely-used-after-if-x-then-return) if you have questions about the redundancy.

```suggestion
          if (board != null) {
            return _build(context, board);
          }
          return const SizedBox.shrink();
```